src/content/docs/code-push/performance.mdx

```diff
@@ -11,23 +11,22 @@ Shorebird uses its own fork of Flutter.
 Our fork works exactly as Google's does, including passing all the same
 functionality and performance tests.
 
-Using Shorebird's fork will result in no change in your App relative to
-Google's. If you believe there is an unintended change, please let us know and
-we'd love to work with you to diagnose.
+Using Shorebird's fork should not result in any change in your app. If you ever
+see any change, please let us know so we can work with you to diagnose!
 
 ### Patching
 
 Updating a Flutter app in production is unique to Shorebird's fork.
 
-Updating a Flutter app with a "patch" will result in the app working exactly as
-well as it does for a "release", just with new code.
+When using Shorebird your app has two different modes of running, un-patched
+("release") and patched ("patch"). Both release and patch builds of your app
+should function identically to as they would without Shorebird. The only caveat
+is on iOS where patched builds can sometimes run slower than release builds due
+to iOS restrictions.
 
-However on iOS our patching mechanism is different from all of the other
-platforms. This is due to restrictions from the Apple App Store.
-
-The Apple App Store requires all updates to use an interpreter. An interpreter
-is a software program that can run other programs (rather than running the
-program directly on the hardware). Interpreters are slow, including ours.
+The iOS App Store requires all app update systems (like Shorebird's) to use an
```

Comment:
Should "requires" be a link back to the App Store guideline just to tie this all together?

Reply:
Added!